packages/shared/lib/core/wallet/utils/transactions/getNonRemainderBasicOrAccountOutputsFromTransaction.ts.ts

```diff
@@ -3,16 +3,15 @@ import { IWrappedOutput } from '@core/wallet/interfaces'
 import { ActivityDirection } from '@core/wallet/enums'
 import { CommonOutput } from '@iota/sdk/out/types'
 
-export function getNonRemainderBasicOutputsFromTransaction(
+export function getBasicOrAccountOutputsFromTransaction(
```

Comment:
it would be better to name both the function and the file the same way :pray: 
```suggestion
export function getNonRemainderBasicOrAccountOutputsFromTransaction(
```

Reply:
Ooops, I forgot to rename the file